service/service.go

```diff
@@ -41,13 +41,13 @@ func (svc *Service) Start() {
 			case <-signals:
 				//Falls into this block when the service is shutdown to safely close the consumer
 
-				svc.Consumer.Closer <- true
-				svc.Producer.Closer <- true
+				svc.Consumer.Closer() <- true
+				svc.Producer.Closer() <- true
 				exitCh <- true
 
 				log.Info("graceful shutdown was successful", nil)
 				return
-			case message := <-svc.Consumer.Incoming:
+			case message := <-svc.Consumer.Incoming():
 
 				instanceID, err := handler.HandleMessage(svc.Producer, svc.S3, svc.ImportAPIURL, message, svc.MaxRetries)
```

Comment:
svc.Producer is `[]byte` but handler takes a `kafka.MessageProducer` instead.

Similar issue with line 59